Modules/readline.c

```diff
@@ -1240,7 +1240,7 @@ static char *
 call_readline(FILE *sys_stdin, FILE *sys_stdout, const char *prompt)
 {
     size_t n;
-    char *p, *q;
+    char *p;
```

Comment:
Why the declaration of `q` has been moved?

Reply:
I like to move variable declaration closer to where they are used. That's now possible thanks to C99 (required by PEP 7 since Python 3.6). @methane and me are moving all variables one by one :-D But only when we already modify something else in the code.

IMHO the main benefit is to see more easily the scope of a variable. Sometimes, it helps to detect that a subfunction should be created.